source/includes/connection-snippets/x509-snippets/python-connection.py

```diff
@@ -0,0 +1,12 @@
+#begin x509 connection
+import ssl
+from pymongo
+import MongoClient
```

Comment:
I'm not familiar with formatting style -- did you mean for a single import to extend to two lines?

Reply:
Typo - fixed in latest commit